content/docs/2.4/scalers/influxdb.md

```diff
@@ -30,18 +30,18 @@ triggers:
 
 **Parameter list:**
 
-- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server. 
+- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server. Can be sourced from `TriggerAuthentication` as well.
```

Comment:
We typically don't add these but only in the Authentication Parameters list below

Reply:
Okay. I was following the azure pipelines scaler as an example.

Reply:
```suggestion
- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server.
```